unit_tests/testStrings/testStrings.f90

```diff
@@ -698,9 +698,9 @@ SUBROUTINE testStrFunct()
   str = '-0.2468e-000'
   ASSERT(str%isFloat(),'with e-000')
   str = '-.2468e-000'
-  ASSERT(.NOT.str%isFloat(),'no leading digit')
+  ASSERT(str%isFloat(),'no leading digit')
   str = '-0.2468-000'
-  ASSERT(.NOT.str%isFloat(),'mixing exponent symbol')
+  ASSERT(str%isFloat(),'missiing exponent symbol')
```

Comment:
Same question here as below... not really sure this is correct...

Also, `missiing` is spelled wrong

Reply:
same answer here......except for missing has more than enough i's.

Reply:
removed missiing's extra i